src/minicraft/entity/mob/MobAi.java

```diff
@@ -100,11 +100,11 @@ public void render(Screen screen) {
 	}
 	
 	@Override
-	public boolean move(int xa, int ya) {
+	public boolean move(int xmov, int yd) {

```

Comment:
This inconsistent variable naming is supremely annoying me. Why is it yd and not ymov?

Reply:
I wonder how that happened